tests/e2e/createnft.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { expect, test } from './fixtures'
+
+const prefix = ['ahk', 'ahp', 'ksm']
```

Comment:
why  have an array if you only use 'ksm'

Reply:
removed